skeleton_for_bot.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import json
+
+TOKEN = 'YOUR_TOKEN' # insert your token from BotFather here
```

Comment:
I think `TOKEN` should be passed via command line arguments, we can use `import argparse`, add argument parsing function and start the application like this:
`python main.py -t <TOKEN> // --token <TOKEN>`